clients/go/pkg/worker/jobWorker_builder.go

```diff
@@ -35,52 +35,57 @@ const (
 	DefaultRequestTimeout         = 10 * time.Second
 )
 
+var defaultBackoffSupplier = NewExponentialBackoffBuilder().Build()
```

Comment:
The other defaults are exported. Not sure whats the value of exporting defaults. Hence I did not export the defaultBackoffSupplier.

Reply:
💭 Not sure either about this. I think this is fine for now, unless someone asks for it.